docker-compose.dev.yml

```diff
@@ -7,22 +7,14 @@ services:
       - '3310:3310'
```

Comment:
Nice update, I would suggest removing `docker-compose.dev.yml` and moving it to `docker-compose.yml`.

Reply:
Sure, you happy for me to remove all the resource constraints? will update readme with this too